cli/src/common_utils/lr-serial-number

```diff
@@ -0,0 +1,22 @@
+if [[ $# -eq 0 ]]; then
+    if [[ -f /opt/seagate/lr-serial-number ]]; then
+        echo "Lyve Rack r1 Serial Number - "`cat /opt/seagate/lr-serial-number`
```

Comment:
Use `$(cat /opt/seagate/lr-serial-number)` instead of \`cat /opt/seagate/lr-serial-number\`